tests/integration/src/test/scala/mesosphere/marathon/integration/ReadinessCheckIntegrationTest.scala

```diff
@@ -125,8 +125,10 @@ class ReadinessCheckIntegrationTest extends AkkaIntegrationTest with EmbeddedMar
       result should be (Created)
 
       And("There is one ongoing deployment")
-      val deployments = marathon.listDeploymentsForBaseGroup().value
-      deployments should have size 1 withClue (s"Expected 1 deployment but found ${deployments}")
+      eventually {
```

Comment:
Once Marathon responds to the above `createAppV2(appV1)` with `Created` the deployment is persisted (saved in memory and ZK) so any subsequent `listDeployments` call should return it. If it is not, this is a **serious violation** of Marathon contract (I don't believe this, so please provide logs to showcase it) and **should not be patched** with `eventually`.